### PR TITLE
brotli,libpng,libjpeg,libwebp,x265,libxml2,xxhash,spirv-tools: disable executable

### PR DESCRIPTION
**Pull request by Andarwinux**
There should be a small speedup of the build. But for future LTO builds, this is a huge speedup and eventually we could get a full ThinLTO build in 60 minutes. (If ffmpeg.exe is disabled and mpv is dynamically linked, it will be reduced to 50 minutes)

### CODE REVIEW

File: packages/x265.cmake [hunk 1]
```diff
@@ -35,6 +35,7 @@ ExternalProject_Add(x265-10bit
         -DBUILD_SHARED_LIBS=OFF
         ${high_bit_depth}
         -DENABLE_SHARED=OFF
+        -DENABLE_CLI=OFF
```
Comment (shinchiro) [inline packages/x265.cmake:38]: Dont disable cli in x265

Comment (Andarwinux) [reply packages/x265.cmake:38]: why?

Comment (shinchiro) [reply packages/x265.cmake:38]: Its useful & may used by many people

Comment (Andarwinux) [reply packages/x265.cmake:38]: But we never released x265.exe, and it's still possible to use x265 via ffmpeg.

**Pull request merged**